hue/hue/base/hue-ini-secret.yaml

```diff
@@ -0,0 +1,179 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hue-ini
+type: Opaque
+stringData:
+  hue.ini: |
+    [desktop]
+      # Hide unused apps
+      app_blacklist=impala,security,jobbrowser,jobsub,pig,hbase,sqoop,zookeeper,spark,oozie,search
+      secret_key=thisISaSECRET_1234
+      http_host=0.0.0.0
+      http_port=8000
+      is_hue_4=true
+      time_zone=America/Los_Angeles
+      django_debug_mode=false
+      dev=false
+      database_logging=false
+      send_dbug_messages=false
+      http_500_debug_mode=false
+
+      [[django_admins]]
+
+      [[custom]]
+
+      [[ldap]]
+        [[[users]]]
+
+        [[[groups]]]
+
+        [[[ldap_servers]]]
+
+      [[vcs]]
+
+      [[database]]
+        engine=django.db.backends.mysql
+        host=hue-mysql.$(namespace).svc
+        port=3306
+        user=$(database_user)
+        password=$(database_password)
+        name=$(database_name)
+
+      [[session]]
+
+      [[smtp]]
+        host=localhost
+        port=25
+        user=
+        password=
+        tls=no
+
+      [[kerberos]]
+
+      [[oauth]]
+
+      [[oidc]]
+
+      [[metrics]]
+
+    [notebook]
+      [[interpreters]]
+        [[[hive]]]
+          name=Hive
+          interface=hiveserver2
+
+        [[[impala]]]
+          name=Impala
+          interface=hiveserver2
+
+        [[[sparksql]]]
+          name=SparkSql
+          interface=hiveserver2
+
+        [[[text]]]
+          name=Text
+          interface=text
+
+        [[[markdown]]]
+          name=Markdown
+          interface=text
+
+    [dashboard]
+      is_enabled=true
+      has_sql_enabled=true
+      has_query_builder_enabled=true
+      has_report_enabled=true
+
+      [[engines]]
+        [[[sql]]]
+        analytics=true
+        nesting=false
+
+        [[[hive]]]
+        analytics=true
+        nesting=true
+
+    [beeswax]
+      hive_server_host=thriftserver.$(namespace).svc
+      hive_server_port=10000
+      auth_username=hue
+      auth_password=password
+      hive_conf_dir=/etc/hive/conf
+
+      [[ssl]]
+
+    [metastore]
+      enable_new_create_table=true
+      force_hs2_metadata=false
+
+    [impala]
+      [[ssl]]
+
+    [spark]
+
+    [oozie]
+
+    [filebrowser]
+
+    [pig]
+
+    [sqoop]
+
+    [proxy]
+
+    [hbase]
+
+    [search]
+
+    [libsolr]
+
+    [indexer]
+
+    [jobsub]
+
+    [jobbrowser]
+
+    [security]
+
+    [zookeeper]
+      [[clusters]]
+        [[[default]]]
+
+    [useradmin]
+      [[password_policy]]
+
+    [liboozie]
+      oozie_url=
+
+    [aws]
+      [[aws_accounts]]
+        [[[default]]]
+          access_key_id=$(s3_key_id)
+          secret_access_key=$(s3_access_key)
```

Comment:
It is better if you name the variables as `s3_access_key_id` and `s3_secret_access_key_id` as they better represent what we are storing in these variables

Reply:
done